pkg/util/k8s/transform.go

```diff
@@ -89,3 +89,14 @@ func TrimPod(obj interface{}) (interface{}, error) {
 	pod.Status.ResourceClaimStatuses = nil
 	return pod, nil
 }
+
+// TrimNode clears unused fields from a Node that are not required by Antrea.
+// It's safe to do so because Antrea only patches Node.
+func TrimNode(obj interface{}) (interface{}, error) {
+	node, ok := obj.(*corev1.Node)
+	if !ok {
+		return obj, nil
+	}
+	node.Status.Images = nil
```

Comment:
is `node.Status.NodeInfo` safe to remove as well?
is it insignificant compared to `node.Status.Images`?

Reply:
It's safe to remove it. However, the field is not a pointer but a struct containing 10 strings, while each string is not too long. To trim them, the simplest code is to override it with a new struct, which could increase heap usage 160 bytes per object temporarily, whille overriding one field per line seems not very worth as each field is not long: the total data size in a typical nodeInfo is about 200 bytes.
```
  nodeInfo:
    architecture: amd64
    bootID: 56ae4e38-a8b1-4657-8b8a-86a75623926e
    containerRuntimeVersion: containerd://1.7.15
    kernelVersion: 5.4.0-169-generic
    kubeProxyVersion: v1.30.0
    kubeletVersion: v1.30.0
    machineID: d61012698a3649178b7f74d6f57fc965
    operatingSystem: linux
    osImage: Debian GNU/Linux 12 (bookworm)
    systemUUID: d43097b6-a686-420f-b033-fd5c15392e4d
```

Reply:
Good point, thanks for the detailed answer